pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -173,7 +173,7 @@ func (mattermost *ClusterInstallation) GenerateIngress() *v1beta1.Ingress {
 }
 
 // GenerateDeployment returns the deployment spec for Mattermost
-func (mattermost *ClusterInstallation) GenerateDeployment(dbUser, dbPassword string, externalDB bool, minioService string) *appsv1.Deployment {
+func (mattermost *ClusterInstallation) GenerateDeployment(dbUser, dbPassword string, externalDB, isLicensed bool, minioService string) *appsv1.Deployment {
```

Comment:
We pass in `isLicensed` here, but don't actually use the value. Should we use that instead on line 314?

Reply:
my brain was melted yesterday, thanks for the catch. replaced